includes/admin/settings/register-settings.php

```diff
@@ -809,6 +838,50 @@ function edd_get_registered_settings() {
 						'type' => 'checkbox',
 					),
 				),
+				'email_summaries' => array(
+					'email_summary_buttons' => array(
+						'id'   => 'email_summary_buttons',
+						'name' => '',
+						'desc' => '<a target="_blank" href="' . esc_url( $email_summary_preview_url ) . '" class="button">' . esc_html( __( 'Preview Email Summary', 'easy-digital-downloads' ) ) . '</a> <a href="' . esc_url( $email_summary_trigger_url ) . '" class="button">'. esc_html( __( 'Send Test Email', 'easy-digital-downloads' ) ) . '</a>',
```

Comment:
When sending a `Test` email can we adjust the subject of the email to contain `[TEST]` so that the user isn't confused by the dates for the summary?

For instance I was a bit confused as to why my date ranges were not starting on my Week Start date until I realized that my report from Today - 7 days until Today. Adding the `[TEST]` to the subject will give the user a clear indication that this is something to test the visual aspects of devilry and what not.